<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SineSettingsWidget</class>
 <widget class="QWidget" name="SineSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>214</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbBasicSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Basic Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <layout class="QVBoxLayout" name="vlBasic">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QLabel" name="lbFreq">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Frequency</string>
            </property>
            <property name="buddy">
             <cstring>leFreq</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leFreq">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <item>
           <widget class="QLabel" name="lbPhase">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Phase</string>
            </property>
            <property name="buddy">
             <cstring>lePhase</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePhase">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbAngular">
            <property name="text">
             <string>Angular</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePiMult">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ï€ /</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePiDiv">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_20">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_26">
          <item>
           <widget class="QLabel" name="lbAmp">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Amplitude</string>
            </property>
            <property name="buddy">
             <cstring>leAmp</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leAmp">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbMin">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Offset</string>
            </property>
            <property name="buddy">
             <cstring>leOffset</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leOffset">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_21">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_27">
          <item>
           <widget class="QLabel" name="lbSampleRate">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Sample Rate</string>
            </property>
            <property name="buddy">
             <cstring>leSampleRate</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSampleRate">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_22">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAdvancedSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Advanced Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_29">
      <item>
       <layout class="QVBoxLayout" name="vlAdvanced">
        <item>
         <widget class="AdvancedSettingsWidget" name="advancedSettings" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">advancedsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leFreq</tabstop>
  <tabstop>lePhase</tabstop>
  <tabstop>leAmp</tabstop>
  <tabstop>leOffset</tabstop>
  <tabstop>leSampleRate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>leFreq</sender>
   <signal>returnPressed()</signal>
   <receiver>SineSettingsWidget</receiver>
   <slot>onLineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhase</sender>
   <signal>returnPressed()</signal>
   <receiver>SineSettingsWidget</receiver>
   <slot>onLineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAmp</sender>
   <signal>returnPressed()</signal>
   <receiver>SineSettingsWidget</receiver>
   <slot>onLineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSampleRate</sender>
   <signal>returnPressed()</signal>
   <receiver>SineSettingsWidget</receiver>
   <slot>onLineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leOffset</sender>
   <signal>returnPressed()</signal>
   <receiver>SineSettingsWidget</receiver>
   <slot>onLineEditReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLineEditReturnPressed()</slot>
 </slots>
</ui>
